polkadot/runtime/parachains/src/inclusion/tests.rs

```diff
@@ -1517,47 +1515,6 @@ fn candidate_checks() {
 			);
 		}
 
-		// candidate not well-signed by collator.
```

Comment:
we could modify this test to check that a badly-signed collator passes the checks

Reply:
done